csrfguard/src/main/java/org/owasp/csrfguard/servlet/JavaScriptServlet.java

```diff
@@ -288,4 +288,4 @@ private void writeJavaScript(final CsrfGuard csrfGuard, final HttpServletRequest
 
         writeJavaScript(request, response);
     }
-}
+}
```

Comment:
Is there a specific reason for this change?

Reply:
It was accidently pushed during the commit. I have removed this file from commits now.